modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/AbstractPageMemoryMvPartitionStorage.java

```diff
@@ -528,20 +516,16 @@ private static byte[] rowBytes(@Nullable BinaryRow row) {
     @Override
     public @Nullable BinaryRow addWrite(RowId rowId, @Nullable BinaryRow row, UUID txId, UUID commitTableId, int commitPartitionId)
             throws TxIdMismatchException, StorageException {
-        assert rowId.partitionId() == partitionId : rowId;
-
-        if (!closeBusyLock.enterBusy()) {
-            throwStorageClosedException();
-        }
+        return busy(() -> {
+            assert rowId.partitionId() == partitionId : rowId;
 
-        try {
             VersionChain currentChain = findVersionChain(rowId);
 
             if (currentChain == null) {
                 RowVersion newVersion = insertRowVersion(row, NULL_LINK);
 
-                VersionChain versionChain = VersionChain.createUncommitted(rowId, txId, commitTableId, commitPartitionId, newVersion.link(),
-                        NULL_LINK);
+                VersionChain versionChain = VersionChain.createUncommitted(rowId, txId, commitTableId, commitPartitionId,
+                        newVersion.link(), NULL_LINK);
```

Comment:
What's the idea behind this change? Old formatting was bad, new one is equally bad. Why bother?